plugins/chat/lib/chat/messages_exporter.rb

```diff
@@ -38,14 +36,17 @@ def get_header(entity)
 
     private
 
+    def oldest_message_date
+      Chat::Message.order(:created_at).pick(:created_at)
```

Comment:
I think performance is not very important in this case, because the next thing we do here is we export all chat messages, which will be incomparably heavier. Though, looks like both versions are more or less the same performance wise: 

1. `order(:created_at).pick(:created_at)` generates a SELECT with ORDER BY and LIMIT 1 which leads to an index scan ([query explanation](https://explain-postgresql.com/archive/explain/736655ab3ec1618170e83d0b58093922:0:2023-07-28))
2. `minimum(:created_at)` generates a query with the MIN aggregate function, which also leads to an index scan ([explanation](https://explain-postgresql.com/archive/explain/e460d3e4da69f76c970b0cf7230318f4:0:2023-07-28))

But, `Chat::Message.maximum(:created_at)` much better translate our intention, so I think we should use it, here is a PR – https://github.com/discourse/discourse/pull/22858.